script/util/rule.js

```diff
@@ -77,7 +77,7 @@ export function rule(filePath) {
 
   while (++index < examples.length) {
     const lines = examples[index].split('\n')
-    /** @type {{name: string, label?: 'input'|'output', setting?: unknown, positionless?: boolean, gfm?: boolean}} */
+    /** @type {{name: string, label?: 'input'|'output', settings?: unknown, setting?: unknown, positionless?: boolean, gfm?: boolean}} */
```

Comment:
this seems like a typo, defining `settings` twice

Reply:
`settings` is the `processor.data('settings', settings)` argument vs. `setting` is the `processor.use(rule, setting)` argument.